consistence-coding-standard.md

```diff
@@ -686,7 +652,7 @@ interface Exception extends \Consistence\Exception
 
 namespace Consistence\Foo;
 
-class LoremException extends \Consistence\PhpException implements \Consistence\Foo\Exception
+class LoremException extends \Consistence\PhpException
```

Comment:
Asi spíš ne, resp. u těch, kde naopak říkám, že se to má jmenovat stejně jako třída atp., tak tam nic takového taky není. Myslím, že když si někdo nepřečte rule, tak komentáře si už nevšimne vůbec :D